seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java

```diff
@@ -252,24 +253,20 @@ private void handleJobInfoById(HttpGetCommand command, String uri) {
         uri = StringUtil.stripTrailingSlash(uri);
         int indexEnd = uri.indexOf('/', URI_MAPS.length());
         String jobId = uri.substring(indexEnd + 1);
-
-        JobInfo jobInfo =
-                (JobInfo)
-                        this.textCommandService
-                                .getNode()
-                                .getNodeEngine()
-                                .getHazelcastInstance()
-                                .getMap(Constant.IMAP_RUNNING_JOB_INFO)
-                                .get(Long.valueOf(jobId));
-
-        JobState finishedJobState =
-                (JobState)
-                        this.textCommandService
-                                .getNode()
-                                .getNodeEngine()
-                                .getHazelcastInstance()
-                                .getMap(Constant.IMAP_FINISHED_JOB_STATE)
-                                .get(Long.valueOf(jobId));
+        IMap<Object, Object> jobInfoMap =
+                this.textCommandService
+                        .getNode()
+                        .getNodeEngine()
+                        .getHazelcastInstance()
+                        .getMap(Constant.IMAP_RUNNING_JOB_INFO);
+        JobInfo jobInfo = (JobInfo) jobInfoMap.get(Long.valueOf(jobId));
+        IMap<Object, Object> finishedJobStateMap =
+                this.textCommandService
+                        .getNode()
+                        .getNodeEngine()
+                        .getHazelcastInstance()
+                        .getMap(Constant.IMAP_FINISHED_JOB_STATE);
+        JobState finishedJobState = (JobState) finishedJobStateMap.get(Long.valueOf(jobId));
```

Comment:
Any reason for change this? Seem like no difference between before.

Reply:
In openEuler system, call /hazelcast/rest/maps/running-job/  returned to  {} ，remote debug finds that the JobState object is not returned correctly. After the code is modified, the JobState object can be returned correctly

![image](https://github.com/apache/seatunnel/assets/39961809/e84af080-186a-4a21-8852-67cb611543a1)
![image](https://github.com/apache/seatunnel/assets/39961809/b92b29af-8b47-4f8a-8492-f7c138d6f359)
![888f2aaf3178c2e301fc309b0f47aec](https://github.com/apache/seatunnel/assets/39961809/71767042-98b3-46f7-8e67-a8f793e9cf93)

Reply:
Feeling a little strange. cc @hailin0 @EricJoy2048